apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerConnectionTests.AbortedTransactionRetryTests.cs

```diff
@@ -183,12 +181,17 @@ public async Task CommitAbortsAlways_RespectsOverallDeadline()
                     var exception = await Assert.ThrowsAsync<SpannerException>(() => connection.RunWithRetriableTransactionAsync(transaction => callee.DatabaseWorkAsync(connection, transaction)));
                     Assert.True(exception.IsRetryable && !exception.SessionExpired);
                     Assert.Contains("Bang!", exception.InnerException.Message);
-
                 });
 
                 // The overall deadline is of 1 hour. The maximum backoff delay is of 32s.
                 // Because of jitter retries can stop at any time after 60mins - 64s, let's give it 2 minutes of range.
                 Assert.InRange(scheduler.Clock.GetCurrentDateTimeUtc(), time0.AddMinutes(58), time0.AddMinutes(60));
+                // The minimum calls that can be made (assuming maximum jitter always) in that time is 60 * 60 / 32 * 2 which is 56.25,
+                // plus 1 because the first one has no delay.
+                // The maximum number of calls tha can be made in 1 hour:
```

Comment:
Nit: tha => that, and on the line below, aprox => approx

Reply:
Done